broccoli/features.js

```diff
@@ -1,35 +1,12 @@
 'use strict';
 
-const fs = require('fs');
-const ts = require('./typescript');
```

Comment:
I see the deprecated features also relies upon this module; I can remove it and merge the deprecated features functionality with this new approach once I sort how the typescript particulars below.

Reply:
Ya, sounds good to me.